### ISSUE TITLE
FXIOS-1852 ⁃ Page Action menu is broken

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#8532]
## Steps to reproduce
1. Go to youtube.com
2. Tap on the Page Action menu
### Actual behavior
- The Page Action menu buttons have a small font size and increasing the large text accessibility size does not increase the font. Also there is a separator line missing between "Send to device" and "Share" menu
### Device & build information
* Device: iPhone Xr (14.4)
* Build version: 34.0 (4847)
### Notes
Attachments: 
![Picture_2021_05_18T12_36_01_214Z](https://user-images.githubusercontent.com/32130829/118651987-c7602b00-b7ee-11eb-997a-75c0b16dd20f.jpeg)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1852)

**Comment by isabelrios** [comment]
Not sure if related, just realized and would need to check on previous builds, but having the device in Spanish those items in the browser tab menu are not localized

**Comment by SimonBasca** [comment]
Verified fix on v34.0 (4879).
